### PR TITLE
🎉 New Source: Grafana

### PR DESCRIPTION
**Pull request by bala-ceg**
## What
[Grafana API](https://grafana.com/docs/grafana/latest/developers/http_api/data_source/)

## Recommended reading order
1. `grafana.yaml`
2. `spec.yaml`

## 🚨 User Impact 🚨
New Feature Request

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

Screenshot 2022-11-02 at 1 22 39 PM

</details>

### CODE REVIEW

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Hello @bala-ceg, Marcos from Airbyte here 👋 . We received more than 25 new contributions along the weekend. One is yours 🎉  thank so much for! Our team is limited and maybe the review process can take longer than expected. As described in the Airbyte's Hacktoberfest your contribution was submitted before November 2nd and it is eligible to win the prize. The review process will validate other requirements. I ask to you patience until someone from the team review it. 

Because I reviewed some contributions for Hacktoberfest so far I saw some common patterns you can check in advance:
* [ ] Make sure you have added connector documentation to `/docs/integrations/`
* [ ] Remove the file `catalog` from `/integration_tests`
* [ ] Edit the `sample_config.json` inside `/integration_tests`
* [ ] For the `configured_catalog` you can use only `json_schema: {}`
* [ ] Add title to all properties in the `spec.yaml`
* [ ] Make sure the `documentationUrl` in the `spec.yaml` redirect to Airbyte's future connector page, eg: connector Airtable the `documentationUrl: https://docs.airbyte.com/integrations/sources/airtable`
* [ ] Review now new line at EOF (end-of-file) for all files.

If possible send to me a DM in Slack with the tests credentials, this process will make easier to us run integration tests and publish your connector. If you only has production keys, make sure to create a bootstrap.md explaining how to get the keys.

**Comment by marcosmarxm** [comment]
Hello! I'm going to be out of the office this Friday and won't be able to review your contribution again today, I return next Monday. So far, most contributions look solid and are almost done to be approved. As said in [Chris' comment](https://airbytehq.slack.com/archives/C0450PCSDB2/p1667430678866629) all contributions made before 2-November are eligible to receive the prize and have 2 weeks to merge the contributions. But I ensure next week we're going to have your contribution merged. If you have questions about the implementation you can send them in `#hacktoberfest-2022` in Airbyte's Slack.

Sorry the inconvenience and see you again next week, thank you so much for your contribution!

**Comment by marcosmarxm** [comment]
@bala-ceg did you check my comments?

**Comment by bala-ceg** [comment]
@marcosmarxm can i please get a day to get back with the changes?

**Comment by marcosmarxm** [comment]
Yes, you can! You have until 15-nov to get this contribution merged. Let me know if you need any assistance.

**Comment by bala-ceg** [comment]
@marcosmarxm I added the missing elements of  the checklist to my recent commit and my PR is ready for review and DM'ed you the test credentials

- [x]  Make sure you have added connector documentation to /docs/integrations/
- [x]  Remove the file catalog from /integration_tests
- [x]  Edit the sample_config.json inside /integration_tests
- [x]  For the configured_catalog you can use only json_schema: {}
- [x]  Add title to all properties in the spec.yaml
- [x]  Make sure the documentationUrl in the spec.yaml redirect to Airbyte's future connector page, eg: connector Airtable the documentationUrl: https://docs.airbyte.com/integrations/sources/airtable
- [x]  Review now new line at EOF (end-of-file) for all files.

**Comment by bala-ceg** [comment]
@marcosmarxm I have only implemented 2 API endpoints that are significant to this connector. 
Can you please suggest any further endpoints?

File: airbyte-integrations/connectors/source-grafana/source_grafana/grafana.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  datasources_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "datasources"
+      path: "/datasources"
+  reports_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "reports"
+      path: "/reports"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-grafana/source_grafana/grafana.yaml:33]: @bala-ceg as described in low-code CDK async endpoints aren't implemented yet. I don't think implementing Grafana using low-code you can reach the goal of this connector. Also I don't see the /report endpoint inlatest version of the API.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
@bala-ceg check my comments.

**Comment by marcosmarxm** [comment]
I don't think this connector API can be implemented by the Airbyte low-code CDK. I'll close the PR feel free to continue the work and show the results @bala-ceg

**Comment by bala-ceg** [comment]
Thanks for reviewing the PR @marcosmarxm, I will work on the other connectors

**Pull request closed without merging**